internal/utility/purl/golang_extractor.go

```diff
@@ -0,0 +1,25 @@
+package purl
+
+import (
+	"log"
+	"strings"
+
+	"github.com/google/osv-scanner/pkg/models"
+)
+
+func ExtractPURLFromGolang(packageInfo models.PackageInfo) (namespace string, name string, ok bool) {
+	nameParts := strings.Split(packageInfo.Name, "/")
+	if len(nameParts) == 0 || len(packageInfo.Name) == 0 {
+		log.Printf("invalid golang package_name=%s", packageInfo.Name)
```

Comment:
same comment here generally: can we return errors instead of just printing and exiting?